src/components/Badge/styles.ts

```diff
@@ -26,13 +26,20 @@ export const Badge = styled('div')<Props>`
   text-transform: ${props => (props.isCapitalized ? 'lowercase' : 'uppercase')};
 
   &:first-letter {
-    text-transform: ${props => props.isCapitalized && 'uppercase'};
+    text-transform: ${props => props.isCapitalized && 'uppercase !important'};
```

Comment:
Aren't you able to do the following instead of using `important`?
`&&& {
   text-transform: ${props => props.isCapitalized && 'uppercase'};
}`

Reply:
The important isn't needed anymore, I forgot to remove it